src/beacon-api/app/routes/devices.py

```
@@ -103,149 +104,130 @@
+        .where(Device.status == 'deployed')
+        .order_by(Device.device_name)
+    ).all()
     
     result = []
-    for row in result_rows:
+    for device in devices:
+        # Get active location
+        location = db.exec(
+            select(Location)
+            .where(Location.device_key == device.device_key)
+            .where(Location.is_active == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Location.is_active:` for truth checks

Replace with `Location.is_active`

(E712)